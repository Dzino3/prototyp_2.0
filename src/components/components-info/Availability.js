import React from 'react'

function Availability(produktInfo) {
  return (
    <div>
      
      <h2 className='avalH22'>{produktInfo.produktInfo.PRODUCT}</h2>
        <div className='orderInfo'>
            <div className='infoSourcedLabel'>
                <h3 className='orderH3'>Availability</h3>
                <label className='reqInfo'>BASE_CODE:</label>
                <label className='reqInfo'>BUNDLE:</label>
                <label className='reqInfo'>BEST_CASE:</label>
                <label className='reqInfo'>WORST_CASE:</label>
                <label className='reqInfo'>COMMENTS:</label>
            </div>
            <div className='infoSourced'>
                <h3 className='orderH3i'>Info</h3>
                <input type='text' name='info' id='BASE_CODEinfo' className='reqInfoOut' placeholder={produktInfo.produktInfo.BASE_CODE} disabled></input>
                <input type='text' name='info' id='BUNDLEinfo' className='reqInfoOut' placeholder={produktInfo.produktInfo.BUNDLE} disabled></input>
                <input type='text' name='info' id='BEST_CASEinfo' className='reqInfoOut' placeholder={produktInfo.produktInfo.BEST_CASE} disabled></input>
                <input type='text' name='info' id='WORST_CASEinfo' className='reqInfoOut' placeholder={produktInfo.produktInfo.WORST_CASE} disabled></input>
                <input type='text' name='info' id='COMMENTSinfo' className='reqInfoOut' placeholder={produktInfo.produktInfo.COMMENTS} disabled></input>
       
            </div>
        </div>
    </div>
  )
}

export default Availability
