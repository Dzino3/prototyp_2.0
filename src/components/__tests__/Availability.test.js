import React from "react";
import renderer from 'react-test-renderer';
import Availability from '../components-info/Availability';

describe(`Rendering Availability`, () => {

    test('should render Availability with Pro, dukt42', function () {
        let produktInfo = {
            produktInfo: "Pro",
            PRODUKT: "dukt42"
        };
        let tree = renderer.create(
            <Availability produktInfo={produktInfo.PRODUKT}></Availability>
        );

        expect(tree.toJSON()).toMatchSnapshot();
    });
})