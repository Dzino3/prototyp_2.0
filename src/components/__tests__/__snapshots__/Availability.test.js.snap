// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rendering Availability should render Availability with Pro, dukt42 1`] = `
<div>
  <h2
    className="avalH22"
  />
  <div
    className="orderInfo"
  >
    <div
      className="infoSourcedLabel"
    >
      <h3
        className="orderH3"
      >
        Availability
      </h3>
      <label
        className="reqInfo"
      >
        BASE_CODE:
      </label>
      <label
        className="reqInfo"
      >
        BUNDLE:
      </label>
      <label
        className="reqInfo"
      >
        BEST_CASE:
      </label>
      <label
        className="reqInfo"
      >
        WORST_CASE:
      </label>
      <label
        className="reqInfo"
      >
        COMMENTS:
      </label>
    </div>
    <div
      className="infoSourced"
    >
      <h3
        className="orderH3i"
      >
        Info
      </h3>
      <input
        className="reqInfoOut"
        disabled={true}
        id="BASE_CODEinfo"
        name="info"
        type="text"
      />
      <input
        className="reqInfoOut"
        disabled={true}
        id="BUNDLEinfo"
        name="info"
        type="text"
      />
      <input
        className="reqInfoOut"
        disabled={true}
        id="BEST_CASEinfo"
        name="info"
        type="text"
      />
      <input
        className="reqInfoOut"
        disabled={true}
        id="WORST_CASEinfo"
        name="info"
        type="text"
      />
      <input
        className="reqInfoOut"
        disabled={true}
        id="COMMENTSinfo"
        name="info"
        type="text"
      />
    </div>
  </div>
</div>
`;
